package.json

```diff
@@ -89,8 +89,6 @@
       "ja",
       "ko",
       "zh",
-      "zh-Hans",
```

Comment:
I'd leave all supported translations here, even if we use some translations for multiple locales. It's easier to handle that in the backend and find out which locales are actually supported. If we have to handle mappings both here and in the backend it becomes harder to manage.

Reply:
From point of translator, maintaining same large translation files is messy. If translation for each region is realy needed, we can create it at the time. Also, mapping here should be set by translation `OWNERS`, I think.

About mapping on backend, we do not need it in new strategy. See my next comment.

Reply:
Translators do not have to manage other translation files as their only step would be to copy/paste it to another directory. We could even prepare a simple script that would keep those files in sync for us. It is also much easier to understand the final structure as every locale would be mapped to the same corresponding directory on the server. No more secret mapping different locales to the same directory.

From developer point of view, it was really confusing to understand whole i18n configuration before as some files contained all supported locales, some only the "actual ones" that we used (directories). I'd prefer to keep all files in sync and have the same list of locales/directories. It introduces small redundancy but is easier to understand.

Reply:
It's not difficult to understand these settings, i.e. for create/update translation files in package.json and set locales for build in angular.json and locale_conf.json. I updated docs also, please see it.